src/linalg/LinAlg.jl

```diff
@@ -1,6 +1,54 @@
 module LinAlg
-include("./graphmatrices.jl")
-end
 
+using SimpleTraits
+using ..LightGraphs
+
+import LightGraphs: IsDirected, adjacency_matrix, laplacian_matrix, laplacian_spectrum
+
+import Base: convert, sparse, size, diag, eltype, ndims, ==, *, .*, issymmetric, A_mul_B!, length, Diagonal
+
+
+export convert,
+	SparseMatrix,
+	GraphMatrix,
+	Adjacency,
+	adjacency,
+	Laplacian,
+	CombinatorialAdjacency,
+	CombinatorialLaplacian,
+	NormalizedAdjacency,
+	NormalizedLaplacian,
+	StochasticAdjacency,
+	StochasticLaplacian,
+	AveragingAdjacency,
+	AveragingLaplacian,
+	PunchedAdjacency,
+	Noop,
+	diag,
+	degrees,
+	symmetrize,
+	prescalefactor,
+    postscalefactor,
```

Comment:
The following lines and previous lines are indented differently

Reply:
Thanks. This was because someone was using tabs instead of spaces. :)